src/content/docs/release-notes/synthetics-release-notes/containerized-private-minions-release-notes/containerized-private-minion-cpm-3082.mdx

```diff
@@ -0,0 +1,13 @@
+---
+subject: Containerized private minion (CPM)
+releaseDate: '2023-04-05'
```

Comment:
@kaschaefer-nr Can you confirm what the release date should be?

Reply:
April 5th. We missed these release notes so they are coming a bit late 😢